pkg/querier/querier.go

```diff
@@ -92,7 +92,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&cfg.MaxSamples, "querier.max-samples", 50e6, "Maximum number of samples a single query can load into memory.")
 	f.DurationVar(&cfg.QueryIngestersWithin, "querier.query-ingesters-within", 0, "Maximum lookback beyond which queries are not sent to ingester. 0 means all queries are sent to ingester.")
 	f.BoolVar(&cfg.QueryStoreForLabels, "querier.query-store-for-labels-enabled", false, "Query long-term store for series, label values and label names APIs. Works only with blocks engine.")
-	f.BoolVar(&cfg.AtModifierEnabled, "querier.at-modifier-enabled", false, "Enable the @ modifier in PromQL.")
+	f.BoolVar(&cfg.AtModifierEnabled, "querier.at-modifier-enabled", true, "Deprecated: Enable the @ modifier in PromQL always.")
```

Comment:
May I ask you to leverage  https://github.com/cortexproject/cortex/blob/master/pkg/util/flagext/deprecated.go  so that this flag increments `cortex_deprecated_flags_inuse_total`

Reply:
added